subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -446,24 +446,34 @@ void *radio_pkt_decrypt_get(void)
 #if !defined(CONFIG_BT_CTLR_TIFS_HW)
 static uint8_t sw_tifs_toggle;
 
-static void sw_switch(uint8_t dir, uint8_t phy_curr, uint8_t flags_curr, uint8_t phy_next,
-		      uint8_t flags_next)
+static void sw_switch(uint8_t dir_curr, uint8_t dir_next,
+		      uint8_t phy_curr, uint8_t flags_curr,
+		      uint8_t phy_next, uint8_t flags_next)
 {
 	uint8_t ppi = HAL_SW_SWITCH_RADIO_ENABLE_PPI(sw_tifs_toggle);
 	uint8_t cc = SW_SWITCH_TIMER_EVTS_COMP(sw_tifs_toggle);
 	uint32_t delay;
 
 	hal_radio_sw_switch_setup(cc, ppi, sw_tifs_toggle);
 
-	if (dir) {
+	if (dir_next) {
 		/* TX */
 
-		/* Calculate delay with respect to current (RX) and next
-		 * (TX) PHY. If RX PHY is LE Coded, assume S8 coding scheme.
+		/* Calculate delay with respect to current and next PHY.
+		 * If RX PHY is LE Coded, assume S8 coding scheme.
```

Comment:
Is the comment still correct? If (dir_curr !=0) then no such assumption is made, but flags_curr is used, right?